app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            "{"
+            + f"Name: {self.name}, "
+              f"Health: {self.health}, "
+              f"Hidden: {self.hidden}"
+            + "}"
+        )
```

Comment:
Use `{{}}` for that purpose instead of `+`

Reply:
flak8 (

Reply:
> Use `{{}}` for that purpose instead of `+`

When I made that and checked with black it made one string which was > 79, so I needed to concatenate strings, that's why it's easier to see logics of concatenation...

Reply:
and as for me, explicit is better than implicit, so when we explicitly add concatenation, it's more obvious to other programs